### PR TITLE
Create a github issue templete with a link to RI2

### PR DESCRIPTION
**Pull request by larskanis**
Hopefully this will lead bug reports to the correct repository.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+This project is for Ruby version 2.3 and older.
```
Comment (luislavena) [inline .github/ISSUE_TEMPLATE.md]: I would emphasize this with `**NOTE**`, `**IMPORTANT**` or similar.

**Review by luislavena — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (larskanis) [inline .github/ISSUE_TEMPLATE.md]: I'll change this. However the markdown syntax is interpreted not before the issue is posted.

**Review by luislavena — APPROVED** [review]
LGTM

**Comment by larskanis** [comment]
@luislavena Thank you for your feedback! Maybe you're interested in the [proposed changes to rubyinstaller.org](https://github.com/oneclick/rubyinstaller.org-website/pull/3) as well?

**Pull request merged**